.readthedocs.yml

```diff
@@ -0,0 +1,10 @@
+version: 2
+
+python:
+  version: "3.8"
```

Comment:
Is it possible instead of hardcoding it here, to reference a value from https://github.com/elastic/rally/blob/cca825568330b66a1ccc09172faba91ecca895ec/.ci/variables.json#L10 (like `MIN_PY_VER`)?

Reply:
I don't see how that would be possible unfortunately, this is just a static file